lib/12T/.vscode/dryrun.log

```diff
@@ -0,0 +1,8 @@
+make --dry-run --always-make --keep-going --print-directory
```

Comment:
I don't think we want to check-in `.vscode` or `.log` file (see earlier comment)

Reply:
agreed - deleted.